services/web-app/pages/about-carbon/how-carbon-works.mdx

```diff
@@ -17,25 +17,25 @@ import { section } from './how-carbon-works.module.scss'
 <AnchorLinks>
   <AnchorLink>The Carbon ecosystem</AnchorLink>
   <AnchorLink>Key terms</AnchorLink>
-  <AnchorLink>Carbon maintainers</AnchorLink>
+  <AnchorLink>Maintainers</AnchorLink>
 </AnchorLinks>
 
 ## The Carbon ecosystem
 
-Carbon is a system of systems. At the epicenter lies the `core system`, with elements, assets, code,
+Carbon is a system of systems. At the epicenter lies the core system, with elements, assets, code,
```

Comment:
One big decision I made while iterating on drafts with our stakeholders was to highlight vocabulary relating to the ecosystem. This increased scannability and helped people adopt the language. I'd like to revert to what we had on that.

Reply:
@aubrey-oneal where else do we use `code syntax highlighting` for non-code emphasis? Wouldn't we use **bold** or _italics_ for copy emphasis instead?

Our inline code snippets come with hover tooltip and copy functionality. Do we want our users copying emphasized words?

<img width="219" alt="image" src="https://user-images.githubusercontent.com/1691245/207642465-f4dfb795-12c3-4e30-a88f-7133862b1540.png">

Per our content guidelines: https://next.carbondesignsystem.com/guidelines/content/writing-style#capitalizing-all-other-words

>If you want to emphasize a particular word or phrase within a sentence, use italic or bold formatting (but not both).

Reply:
We use highlighting in our tutorials for vocabulary. For example: https://next.carbondesignsystem.com/designing/figma
<img width="747" alt="Screenshot 2022-12-14 at 1 07 03 PM" src="https://user-images.githubusercontent.com/47198241/207690414-2f5ed2d4-7eeb-434a-9d35-a8b973f0ce56.png">

While I prefer the visual style of the highlights, now that I know it comes with tooltip functionality it does seem a bit odd. Maybe we keep it as is and do a site-wide change at some point to either bold or italics.

Reply:
@aubrey-oneal is bold okay? I'd like to adhere to our writing style guidelines. See below. It's awkward to have hover tooltips and click copy-to-clipboard functionality when those emphasized words are formatted as code snippets.

<img width="2560" alt="image" src="https://user-images.githubusercontent.com/1691245/207733909-ed2f7f4f-3aec-412e-9a62-05828dc74bc6.png">

Reply:
Is this style of highlighting shown on the homepage available to use elsewhere?

<img width="814" alt="image" src="https://user-images.githubusercontent.com/11670886/207738801-26961106-e201-48b7-a9e0-3cbab44f2335.png">

Reply:
On the kit page we use the code snippet/highlighter to indicate exact names of libraries used in the Figma tool, its not just an emphasis.

Reply:
Right now, the `<mark>` element styling is scoped to that home page intro paragraph.

We could make it a global style, and make it work with both normal body copy and that larger copy.

<img width="639" alt="image" src="https://user-images.githubusercontent.com/1691245/207739658-901d1de0-c810-445c-85ef-d9ae7e954e56.png">

The only challenge would be how to handle the trailing comma. I don't know how we'd give the emphasis highlight left and right padding without covering the trailing comma.

Reply:
So it's not available outside of the home page, and the effort would be low if we wanted to make that a global style to use in other places.

Reply:
Nevermind - we could make it work with trailing comma 😎.

<img width="626" alt="image" src="https://user-images.githubusercontent.com/1691245/207740100-5586867c-f4d5-4902-8da8-41cd1e7d9542.png">

Reply:
My recommendation is we stick with bold for now to follow our content guidelines, and if we want to use "mark" highlights across the site, we do that in a separate issue.

Reply:
Yeah you're probably right, we'd want to be intentional/consistent with adding that highlight as well. I know we do bolding emphasis elsewhere in the docs so I think its good to stick with that for now.